src/NServiceBus.Hosting.Windows/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="CommonServiceLocator" version="1.0" targetFramework="net45" />
-  <package id="Fody" version="1.26.1" targetFramework="net45" developmentDependency="true" />
+  <package id="CommonServiceLocator" version="1.3" targetFramework="net45" />
```

Comment:
Tests were failing, but I'll have another go. Had quite a few problems getting it properly compiled the first time. Talked with Simon about Fody which I needed to upgrade as well. But found out later a reference with ServiceLocator was bad. I'll downgrade and see if it'll work.